src/run.rs

```diff
@@ -7,8 +7,21 @@ use crate::{instance::ExitCondition, utils::Error, Instance, RunOptions};
 const DEFAULT_PROGRAM_NAME: &str = "wasm";
 
 /// Run a WASIX program.
-#[wasm_bindgen]
-pub async fn run(wasm_module: WasmModule, config: RunOptions) -> Result<Instance, Error> {
+///
+/// # WASI Compatibility
+///
+/// The WASIX standard is a superset of [WASI preview 1][preview-1], so programs
+/// compiled to WASI will run without any problems.
+///
+/// [WASI Preview 2][preview-2] is a backwards incompatible rewrite of WASI
+/// to use the experimental [Component Model Proposal][component-model]. That
+/// means programs compiled for WASI Preview 2 will fail to load.
+///
+/// [preview-1]: https://github.com/WebAssembly/WASI/blob/main/legacy/README.md
+/// [preview-2]: https://github.com/WebAssembly/WASI/blob/main/preview2/README.md
+/// [component-model]: https://github.com/WebAssembly/component-model
+#[wasm_bindgen(js_name = "runWasix")]
+pub async fn run_wasix(wasm_module: WasmModule, config: RunOptions) -> Result<Instance, Error> {
```

Comment:
After thinking a bit about this. I think we can have a "WASIXRunner" that takes a packageCommand and runs it with the config? (in a more abstract way). That way we prepare when we have more runners. I'm not religious on this now though, just some food for thought

Reply:
I deliberately don't want to go down that path. 

The `run()` function is meant to be an easy way for someone to run a WASI/WASIX program without needing to opt into the entire Wasmer ecosystem. That's why it doesn't mention anything about commands or packages or runners or any of that.

Reply:
SGTM